.github/workflows/build-devcontainer.yaml

```diff
@@ -66,8 +56,9 @@ jobs:
           push:
             ${{ github.ref == 'refs/heads/main' || startsWith(github.ref,
             'refs/tags/') }}
+          # As above re `prql`
           cache-from: |
-            ghcr.io/${{ github.repository_owner }}/prql-devcontainer-base
+            ${{ env.DOCKER_METADATA_OUTPUT_TAGS }}
```

Comment:
Great idea, I think this will work! https://github.com/PRQL/prql/actions/runs/5233442749/jobs/9448918144?pr=2786#step:5:10